src/Pages/Result/Results.jsx

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable camelcase */
+import React from 'react';
+import { Link } from 'react-router-dom';
+import Styles from './Results.module.css';
+import { QuestionConsumer } from '../../Context';
+
+export default function Result() {
+  return (
+    <QuestionConsumer>

```

Comment:
use hooks instead

Reply:
This was just to understand the use of react context